S24-testing/15-done-testing.t

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env raku
+use Test;
+use Test::Util;
```

Comment:
I wonder how does it work for you? For this `use` to work you need this line to precede it:

```raku
use lib $?FILE.IO.parent(2).add: 'packages/Test-Helpers';
```

Reply:
RAKUDOLIB=~/Repo/roast/packages/Test-Helpers/lib prove6 -v 15-done-testing.t

Reply:
I see. A little advise for the future. Roast is used to test freshly-compiled rakudo compiler. For this reason a test in roast must be possible to run using something like `./rakudo-m -Ilib t/spec/<S-dir>/<testname>` executed in the build dir. Also it must be runnable with `perl t/harness<5 or 6>`.